docs/cicd-with-bicep.md

```diff
@@ -28,12 +32,12 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      # Install the latest release of the bicep CLI
-      - name: Install bicep CLI
+      # Install the latest release of the bicep CLI binaries
+      - name: Install bicep CLI Binaries
         run: |
-          curl -Lo bicep https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64
-          chmod +x ./bicep
-          sudo mv ./bicep /usr/local/bin/bicep
+          curl -Lo bicep.bin https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64
```

Comment:
@anthony-c-martin - any idea why @pinakighatak needed to add the `.bin` extension here?

Reply:
Any extension will work. I just put bin. Curl does not seem to work without this extension.

Reply:
I'm not sure - I've not had any problems running this command on OSX, Linux or WSL.

@pinakighatak Does the following work for you by any chance (specifying the current directory with `./`)?
```
curl -Lo ./bicep https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64
```

It doesn't matter much, as this is just a temporary name before the exe is moved to `/usr/local/bin/bicep`

Reply:
I'm trying this directly on my GitHub workflow pipeline. Adding ./bicep does not work. See below:
![image](https://user-images.githubusercontent.com/57876192/111325940-71df9480-866c-11eb-848a-5de4c3774aa6.png)

But adding an extension (any) works smoothly.
![image](https://user-images.githubusercontent.com/57876192/111327085-5e80f900-866d-11eb-9921-983962d86e4f.png)

Reply:
given the extension doesn't matter, I'm inclined to merge this.